CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 ## v0.4.0 (2022-01-19)
 
-  * Update Sass version to `1.49.0`
+  * Update Sass version to `1.49.11`
```

Comment:
I am assuming this just got caught by search/replace, but these changes will be part of the next release, so we should not change the version here :)

```suggestion
  * Update Sass version to `1.49.0`
```